File: calio-us-berkeley/2024/problems/diceroll/templates/diceroll_template.py
def solve(N: int, X: list[int]) -> int:
    """
    Return the best number to "zero out" in order to
    minimize the expected value of a dice roll.
    
    N: the number of faces on the die
    X: the list of numbers on each face of the die
    """
    # YOUR CODE HERE
    return 0


def main():
    T = int(input())
    for _ in range(T):
        N = int(input())
        X = [int(x) for x in input().strip().split(' ')]
        print(solve(N, X))

if __name__ == '__main__':
    main()

File: calio-us-berkeley/2024/problems/diceroll/templates/diceroll_template.cpp
#include <iostream>
#include <vector>

using namespace std;

/**
 * Return the best number to "zero out" in order to
 * minimize the expected value of a dice roll.
 * 
 * N: the number of faces on the die
 * X: the list of numbers on each face of the die
 */
int solve(int N, vector<int> X) {
    // YOUR CODE HERE
    return -1;
}

int main() {
    int T;
    cin >> T;
    for (int i = 0; i < T; i++) {
        int N;
        cin >> N;
        vector<int> X(N);
        for (int &item : X) {
            cin >> item;
        }
        cout << solve(N, X) << '\n';
    }
}
